components/docs/Container.vue

```diff
@@ -110,6 +110,9 @@
     font-weight: 600;
     line-height: 2.375;
 }
+:deep(img){
+    width: 100%;
+}
```

Comment:
This will lead to this issue (I've already face it 😄)

![](https://i.gyazo.com/571e9dc2e7bd5e1a8500172e059580a9.png)

I'm thinking another solution. Feel open to share your thoughts.

Reply:
thanks, They are using same component